ecs-cli/Gopkg.toml

```diff
@@ -0,0 +1,66 @@
+
+# Gopkg.toml example
```

Comment:
Should this example code be removed?

Reply:
i think the examples here are useful since there are a few ways to constrain your dependencies.